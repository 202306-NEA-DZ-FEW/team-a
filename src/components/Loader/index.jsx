import Image from "next/image";
import Spinner from "public/images/spinner.svg";

function Loader() {
  return (
    <div className='flex justify-center items-center h-screen'>
      <Image src={Spinner} alt='loading' height={100} width={100} priority />
    </div>
  );
}

export default Loader;
