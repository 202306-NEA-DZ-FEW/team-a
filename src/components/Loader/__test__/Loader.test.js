import renderer from "react-test-renderer";

import Loader from "..";

it("renders correctly", () => {
  const tree = renderer.create(<Loader />).toJSON();
  expect(tree).toMatchSnapshot();
});
