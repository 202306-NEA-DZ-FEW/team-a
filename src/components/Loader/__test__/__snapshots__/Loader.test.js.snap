// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="flex justify-center items-center h-screen"
>
  <img
    alt="loading"
    data-nimg="1"
    decoding="async"
    fetchpriority="high"
    height={100}
    onError={[Function]}
    onLoad={[Function]}
    src="/_next/image?url=%2Fimg.jpg&w=256&q=75"
    srcSet="/_next/image?url=%2Fimg.jpg&w=128&q=75 1x, /_next/image?url=%2Fimg.jpg&w=256&q=75 2x"
    style={
      Object {
        "color": "transparent",
      }
    }
    width={100}
  />
</div>
`;
